py/bitbox02/devices.py

```diff
@@ -34,3 +37,11 @@ def get_bitbox02_devices(product_string=BITBOX02):
         and (info["usage_page"] == 0xFFFF or info["interface_number"] == 0)
         and info["product_string"] == product_string
     ]
+
+
+def parse_device_version(serial_number):
```

Comment:
I like that the regexp works no matter what comes before (e.g. it will work if we strip the prefix), and that it works robustly also with the bootloader version. It happens to have the same length prefix (`bl` instead of `fw`), but that sounds pretty fragile.

I will extend the regexp to go to the end of the line instead if ok.

Reply:
Done.

Reply:
regexes can also be hard. It has to be `.*` as there might not be any more characters after the last number, and it should end with `$` so that it takes the whole string until the end. `r"v([0-9]+\.[0-9]+\.[0-9]+.*$)"`

Reply:
:flushed: fixed